packages/block-editor/src/components/global-styles/color-panel.js

```diff
@@ -715,7 +715,7 @@ export default function ColorPanel( {
 			{ items.map( ( item ) => (
 				<ColorPanelDropdown
 					key={ item.key }
-					{ ...item }
+					{ ...omit( item, [ 'key' ] ) }
```

Comment:
Instead of making `omit` a shared utility, can we instead remove it and just do:

```
const { key, ...restItem } = item;
```

and then spreading `{ ...restItem }` instead?